migrations/000079_ExportImportTraveler.up.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2021 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE exportimport
+  ADD COLUMN traveler BOOL DEFAULT false;
+ALTER TABLE exportimport
```

Comment:
I think this will either need to be a separate transaction or you need to do a data migration here. Setting a column as a default doesn't backfill existing rows.

```sql
ALTER TABLE add column;
UPDATE exportimport SET traveler = false;
ALTER COLUMN traveler SET NOT NULL;
```